cheri_compressed_cap.h

```diff
@@ -93,6 +98,25 @@ enum CC256_OTypes {
     CC256_SPECIAL_OTYPE(LAST_NONRESERVED_OTYPE, 15),
 };
 
+#define LS_SPECIAL_OTYPES_COMMON(ITEM, ...)                                                                            \
+    ITEM(OTYPE_UNSEALED, __VA_ARGS__)                                                                                  \
+    ITEM(OTYPE_SENTRY, __VA_ARGS__)
+
+#ifdef CC_IS_MORELLO
+
+#define LS_SPECIAL_OTYPES(ITEM, ...)                                                                                   \
```

Comment:
Just always use the non-Morello names and remove the need for this?

Reply:
Given that we have not implement the RISCV ones, they should probably be separate until such a time we confident they are the same.